meta-iot2000-example/recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -1,5 +1,23 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/swupdate:"
 
-SRCREV = "abad107582ab6230af3eb74d15227a1daa699e6d"
+SRCREV = "012374087c747f05216a7f002e4b8a34bc142c52"
+
+SRC_URI += "file://swupdate_handlers.lua"
+SRC_URI += "file://tools_makefile.patch"
+SRC_URI += "file://progress_firmware.c"
+
+FILES_${PN} += "/etc/swupdate.cfg /usr/bin/progress_firmware"
 
 DEPENDS += "efibootguard"
+
+do_configure_prepend () {
+        cp ${WORKDIR}/swupdate_handlers.lua ${S}
+        cp ${WORKDIR}/Makefile ${S}/tools/Makefile
+        cp ${WORKDIR}/progress_firmware.c ${S}/tools/progress_firmware.c
+}
+
+do_install_append() {
+        install -d ${D}${sysconfdir}
+        install -m 644 ${WORKDIR}/swupdate.cfg ${D}${sysconfdir}/swupdate.cfg
+        install -m 755 ${S}/tools/progress_firmware_unstripped ${D}/usr/bin/progress_firmware
```

Comment:
Why the unstripped variant?

Reply:
Ah I guess that is a relict from the unneeded -g option for C we had in the config before.

Reply:
If bitbake strips it, you cannot sensibly debug anymore. So, it's a matter of correctly feeding bitbake's assumptions. I would prefer to selectively skip QA here or make a note why this is here for clarity. As is, one could also think of deploying unstripped binaries to a production target image...

Reply:
Okay I add an insane_skip